include/control_toolbox/pid.hpp

```diff
@@ -253,11 +220,11 @@ class Pid
    * and the timestep \c dt.
    *
    * \param error  Error since last call (error = target - state)
-   * \param dt Change in time since last call
+   * \param dt Change in time since last call in seconds
    *
    * \returns PID command
    */
-  double computeCommand(double error, rclcpp::Duration dt);
+  double computeCommand(double error, double dt);
```

Comment:
not sure if double makes sense here. if it's clearly in seconds, then I would assume something unsigned. If higher precision, why not using nanoseconds (`uint64_t`)?